### DISCUSSION TITLE
Abfrageergebnisse unterscheiden sich zwischen Massen- und Einzelabfrage

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#208] [Q&A]
Wenn ich in der Schnellsuche die Grundstücksnummer ".1" eintrage, erhalte ich 100 Treffer (alle Grundstücke mit der Nummer .1 in Vorarlberg). Klicke ich dann auf ein Ergebnis, werden in den Eigenschaften u.a. Inhalte angezeigt, welche ich im CMS auf "nicht sichtbar" gestellt habe (also das Häkchen bei "Sichtbar" entfernt wurde).
Wenn ich in der Schnellsuche das gesuchte Grundstück auf ".1 Gemeindename" einschränke (also nur noch 1 Treffer erhalte, statt 100) werden diese unsichtbaren Eigenschaften **nicht** dargestellt.
Die Suche ist mit der GIS-Suche realisiert (mit der .json Datei im etc Ordner).
Das Problem ist wahrscheinlich schlecht nachzuvollziehen, weshalb ich eine kurze Videokonferenz vorschlage.

**Comment by gis-eni** [comment]
Ok, dieses Problem kann ich nachvollziehen und es sollte mit den kommenden Build >= 5/6.24.4202 behoben sein.
Da es sich nicht um einen sicherheitsrelevanten Fehler handelt, kommt diese Build erst nach der Behebung anderer Bugs.

Frage: Warum setzt du Felder auf "nicht sichtbar"?
